### ISSUE TITLE
[Crash]: 

### ISSUE THREAD

**Issue by Kozello** [Fuzss/puzzleslib#23]
### Mod Loader (Required)

Fabric

### Minecraft Version (Required)

1.19.2

### Mod Version (Required)

4.3.12

### Notes (Required)

Whenever i launch fabric with this mod the game crashes

### Crash Report (Required)

https://paste.gemwire.uk/view/4ec3503e

**Comment by Fuzss** [comment]
```
info: [17:29:30] [main/WARN]: Mod resolution failed
info: [17:29:30] [main/INFO]: Immediate reason: [HARD_DEP_NO_CANDIDATE enchantinginfuser 4.2.2 {depends puzzleslib @ [>=4.3.12]}, ROOT_FORCELOAD_SINGLE enchantinginfuser 4.2.2]
info: [17:29:30] [main/INFO]: Reason: [HARD_DEP enchantinginfuser 4.2.2 {depends puzzleslib @ [>=4.3.12]}]
info: [17:29:30] [main/INFO]: Fix: add [add:puzzleslib 4.3.12 ([[4.3.12,?)])], remove [], replace []
```

**Comment by Kozello** [comment]
yea i know i put the puzzleslib and it still crashes
ill send the new log
